bower.json

```diff
@@ -7,8 +7,8 @@
     "ember-inflector": "1.3.1",
     "ember-mocha": "~0.8.11",
     "Faker": "3.0.1",
-    "lodash": "~4.5.0",
-    "perfect-scrollbar": "~0.6.10",
+    "lodash": "^4.10.0",
+    "perfect-scrollbar": ">=0.6.7 <2.0.0",
```

Comment:
Please - also, if you can raise an issue against `ember-frost-core` to adjust that dependency range I would appreciate it